bpf/gotracer/go_nethttp.c

```diff
@@ -803,16 +807,26 @@ static __always_inline void setup_http2_client_conn(void *goroutine_addr, void *
         }
 
 #ifndef NO_HEADER_PROPAGATION
+        void *framer = 0;
+        // TODO: Add this offset
```

Comment:
Needs to be removed as comment, I added the offset. I'll remove it once I fix 2) for HTTP2.